frontend/src/components/SignUpForm/SignUpForm.jsx

```diff
@@ -9,7 +9,7 @@ function SignUpForm (){
     const [firstName, setFirstName] = useState("");
     const [lastName, setLastName] = useState("");
 
-    // const { updateUser } = useContext(UserContext);
+    const { updateUser } = useContext(UserContext);
     const navigate = useNavigate();
 
     const handleOnSubmit = async (event) => {
```

Comment:
You should check that the fields are all not empty before fetching the backend.
Also some validation for password and email would be good.